src/cli/commands/lint.ts

```diff
@@ -1,20 +1,14 @@
 import { Command, flags as flagHelpers } from '@oclif/command';
 import { IParserResult } from '@stoplight/types';
-import { getLocationForJsonPath, parseWithPointers } from '@stoplight/yaml';
-import { existsSync, readFileSync, writeFile } from 'fs';
+import { getLocationForJsonPath } from '@stoplight/yaml';
 import { resolve } from 'path';
-import { promisify } from 'util';
-
-// @ts-ignore
-import * as fetch from 'node-fetch';
-
-import { json, stylish } from '../../formatters';
+import { readRuleset } from '../../config/reader';
```

Comment:
Is the folder /config/ maybe a layover from the old naming? Can this be rulesetReader or something?

Reply:
Nope, I intentionally left it in `config` folder. My opinion is that loading rulesets is part of "configuring" spectral. The `reader` file can be extended with `function readConfigFile(` for instance. 

We don't really follow any file naming convention at the moment so both `rulesetReader/read` and `config/readRuleset` are equally good but judgemental.

Reply:
Conceptually rulesets and config have no relationship or connection to each other at all. Config is just another form of input like CLI options. Please can we keep the two ideas separate.

Reply:
will move everything currently stored in `config/` folder into `ruleset/` folder.
I want to keep it separate from `rulesets/` (plural) since the latter is used for storing actual rulesets (oas2/3) , rather than ruleset loading logic

Reply:
done `rulesets`